tests/internal/test_fio_fillup.py

```diff
@@ -0,0 +1,62 @@
+"""
+Module to perform IOs with several weights
+"""
+import pytest
+import logging
+
+from ocs_ci.framework.testlib import BaseTest
+from ocs_ci.ocs import constants
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestPVCFillup(BaseTest):
+    """
+    Test PVC Fillup
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["size", "percentage"],
+        argvalues=[pytest.param(*['10', '50'])]
+    )
+    def test_fillup_fs(self, size, percentage,
+                       teardown_factory, storageclass_factory, interface_iterate,
```

Comment:
you don't need to use parametrize for one test case.
you can change:
def test_fillup_fs(self, size='10', percentage='50',....)

Reply:
@OdedViner This is true, but i used parameterize in case we would run with different set of parameters.